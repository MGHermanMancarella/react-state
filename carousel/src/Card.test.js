import React from 'react'
import { render } from '@testing-library/react'
import Card from './Card.js'

//smokeTests for Cards
it('renders without crashing', function () {
  // this is a low-value test, but better than nothing
  render(<Card caption='test' src='test' currNum='0' totalNum='1' />)
})

// end

it('matches snapshot', function () {
  const { container } = render(
    <Card caption='test' src='test' currNum='0' totalNum='1' />
  )
  expect(container).toMatchSnapshot()
})
// end
