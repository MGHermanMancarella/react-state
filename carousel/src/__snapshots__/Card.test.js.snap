// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      test
    </h4>
    <img
      alt="test"
      class="Card-image"
      src="test"
    />
    <small
      class="Card-small"
    >
      Image 
      0
       of 
      1
      .
    </small>
  </div>
</div>
`;
